### PR TITLE
Build UMD modules in root['cozy']

### PR DESCRIPTION
**Pull request by m4dz**
This PR ensure we build a real UMD module, and exposes it at root level under the namespace `cozy` (i.e. `window.cozy.client`)

### CODE REVIEW

**Comment by nono** [comment]
@m4dz can you fix the build? In the unit test, there are `cozy = new Cozy` that should be replaced with `cozy = new Client` (and maybe other errors)

File: src/index.js [hunk 1]
```diff
@@ -244,12 +245,6 @@ function addToProto (ctx, obj, proto, disablePromises) {
...
-export default cozy
-export { Cozy, LocalStorage, MemoryStorage }
-
-if ((typeof window) !== 'undefined') {
-  window.cozy = cozy
-  window.Cozy = Cozy
-}
+module.exports = new Client()
+module.exports.LocalStorage = LocalStorage
+module.exports.MemoryStorage = MemoryStorage
```
Comment (enguerran) [inline src/index.js]: If we use `babel-loader`, why can't we use es2015 `import/export`?

Comment (m4dz) [inline src/index.js]: We can't, as webpack expect `module.exports` (so, a commonjs format) to build proper UMD. Else, it won't output a `root['cozy']['client'] = Client` but `root['cozy']['client'] = {default: Client}` which isn't what we expect :wink:

Comment (enguerran) [inline src/index.js]: understood

**Comment by m4dz** [comment]
@nono done, thanks :tada:

**Comment by nono** [comment]
@m4dz does the documentation needs to be updated?

**Comment by nono** [comment]
Conflicts resolved in #75

**Pull request merged**